index.js

```diff
@@ -4,22 +4,40 @@
  * @param {Object} store - An object representing a single store. See the instructions for details on its shape.
  * @returns {Object[]} The same `stores` array that was inputted.
  */
-function addNewStore(stores, store) {}
+function addNewStore(stores, store) {
+  stores.push(store)
+  return stores
+}
 
 /**
  * Removes a store object at the given position.
  * @param {Object[]]} stores - An array of store objects.
  * @param {number} index - A number representing the index of the store to be removed from the array.
  * @returns {Object[]} The same `stores` array that was inputted.
  */
-function removeStoreAtPosition(stores, index) {}
+function removeStoreAtPosition(stores, index) {
+   if (index ===0){
+    stores.shift()
+   }
+   else{
+  stores.splice(index,index)}
```

Comment:
Careful with this, doing `splice(index, index)` means that if the index were 2, we'd be removing 2 items starting at position 2, or if index was 5, we'd be removing 5 items starting from index 5.

Reply:
Okay, I see. I think I may have misread the MDN docs.  Thank you Steven!